client/my-sites/stats/info-panel/index.jsx

```diff
@@ -19,26 +19,25 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
-		debug('Rendering stats info panel', this.props );
 		var infoView,
-			infoPanelClass = "module-content-text module-content-text-info";
+			infoPanelClass = 'module-content-text module-content-text-info';
 
-		switch( this.props.module ) {
+		switch ( this.props.module ) {
 
 			case 'referrers':
 				infoView = (
 					<div className={ infoPanelClass }>
 						<p>{ this.translate( 'Learn more about your site’s visibility by looking at the websites and search engines that send the most traffic your way.' ) }</p>
-						<ul className="documentation">
+						<ul className='documentation'>
```

Comment:
Just a follow-up comment to note that we discussed these in slack and we're going to be taking care of these as they come up as well as some es6 formatting. I had overzealously fixed some double-quote warnings I was getting so now we're getting single-quote warnings :)